model/src/test/java/oracle/kubernetes/weblogic/domain/v2/BaseConfigurationTestBase.java

```diff
@@ -84,4 +84,16 @@ public void whenServerStartPoliciesDiffer_objectsAreNotEqual() {
   private V1EnvVar env(String name, String value) {
     return new V1EnvVar().name(name).value(value);
   }
+
+  @Test
```

Comment:
The names of tests don't affect their functionality, but they are very useful for explaining what is going on to the next developer. Note that most test names have statements that tell you what is being tested.

Also, each case should test a single functionality. That makes naming the tests easy. See the other tests in this class.

Reply:
Looks good